grafana/README.md

```diff
@@ -7,4 +7,60 @@ Status notify: we are still working on this config. It's expected to change freq
 
 # How to use
 
+## `greptimedb.json`
+
 Open Grafana Dashboard page, choose `New` -> `Import`. And upload `greptimedb.json` file.
+
+## `greptimedb-cluster.json`
+
+This cluster dashboard provides a comprehensive view of incoming requests, response statuses, and internal activities such as flush and compaction, with a layered structure from frontend to datanode. Designed with a focus on alert functionality, its primary aim is to highlight any anomalies in metrics, allowing users to quickly pinpoint the cause of errors.
+
+We use Prometheus to scrape off metrics from nodes in GreptimeDB cluster, Grafana to visualize the diagram. Any compatible stack should work too.
+
+__Note__: This dashboard is still in an early stage of development. Any issue or advice on improvement is welcomed.
+
+### Configuration
+
+Please ensure the following configuration before importing the dashboard into Grafana.
+
+__1. Prometheus scrape config__
+
+Assign `greptime_pod` label to each host target. We use this label to identify each node instance.
+
+```yml
+# example config
+# only to indicate how to assign labels to each target
+# modify yours accordingly
+scrape_configs:
+  - job_name: metasrv
+    static_configs:
+    - targets: ['<ip>:<port>']
+      labels:
+        greptime_pod: metasrv
+
+  - job_name: datanode
+    static_configs:
+    - targets: ['<ip>:<port>']
+      labels:
+        greptime_pod: datanode1
```

Comment:
Does it matter if we have more than 3 datanodes?

Reply:
Just add `greptime_pod` label to each datanode should do. We don't limit the number of node. Same to frontend and metasrv.